test/wrapper_test.py

```diff
@@ -874,11 +870,14 @@ def _mime_vs_type(mime, requested_type):
         self.assertEqual(0, _mime_vs_type("text/n3", N3))
         self.assertEqual(0, _mime_vs_type("text/turtle", TURTLE))
         self.assertEqual(0, _mime_vs_type("application/turtle", TURTLE))
-        self.assertEqual(0, _mime_vs_type("application/ld+json", JSON))  # Warning
-        self.assertEqual(0, _mime_vs_type("application/ld+json", JSONLD))  # Warning
-        self.assertEqual(0, _mime_vs_type("application/rdf+xml", XML))  # Warning
-        self.assertEqual(0, _mime_vs_type("application/rdf+xml", RDF))  # Warning
-        self.assertEqual(0, _mime_vs_type("application/rdf+xml", RDFXML))  # Warning
+        self.assertEqual(0, _mime_vs_type("application/json", JSON))  # Warning
+
+        # graph.load() is deprecated, it will be removed in rdflib 6.0.0. Please use graph.parse() instead.
+        self.assertEqual(1, _mime_vs_type("application/ld+json", JSONLD))  # Warning
```

Comment:
One of the three test failures is because this is returning no errors. Why is this returning none where as XML below returns 1?

Reply:
Actually, I don't know. In my local environment, `test/wrapper_test.py` isn't failing. But it is failing in CI environment...

```shellsession
$ python test/wrapper_test.py
.........s...................../Users/eggplants/prog/sparqlwrapper/SPARQLWrapper/Wrapper.py:763: RuntimeWarning: Sending Accept header '*/*' because unexpected returned format 'json-ld' in a 'SELECT' SPARQL query form
  warnings.warn(
....
----------------------------------------------------------------------
Ran 35 tests in 0.124s

OK (skipped=1)
```